server/events.go

```diff
@@ -832,14 +832,55 @@ func (s *Server) leafNodeConnected(sub *subscription, _ *client, subject, reply
 	}
 }
 
+// Common filter options for system requests STATSZ VARZ SUBSZ CONNZ ROUTEZ GATEWAYZ LEAFZ
+type ZFilterOptions struct {
+	Name    string `json:"server"`
+	Cluster string `json:"cluster"`
+}
+
+func (s *Server) filterRequest(msg []byte) (bool, error) {
```

Comment:
Seems that this returns `true` if the server should be filtered *out*. So maybe rename or add some comment on what that means if this returns `true` (or `false`).

Reply:
@kozlovic added the comment & squashed